mods/cnc/languages/chrome/en.ftl

```diff
@@ -330,7 +330,7 @@ button-top-buttons-repair-tooltip = Repair
 button-top-buttons-beacon-tooltip = Place Beacon
 button-top-buttons-options-tooltip = Options
 button-production-types-building-tooltip = Buildings
-button-production-types-defence-tooltip = Support
+button-production-types-defense-tooltip = Support
```

Comment:
If this is going to be renamed it should IMO go to support.

Reply:
split into #21584